bin/i18n-codeorg/in.sh

```diff
@@ -74,3 +74,7 @@ loc_dir=i18n/locales/source/markdown/public/educate/curriculum/
 mkdir -p $loc_dir
 orig_file=pegasus/sites.v3/code.org/public/educate/curriculum/csf-transition-guide.md
 cp_in $orig_file $loc_dir
+loc_dir=i18n/locales/source/markdown/public/
```

Comment:
You probably don't need to do this because this directory gets created by a couple of the lines above.

Reply:
Yeah its redundant, but I thought it would be nice to have because its a dependency of this markdown file. I'm doing this to try and protect against someone removing the mkdir above without realizing its needed for stuff below.

Reply:
Sounds good!